src/conversion.rs

```diff
@@ -319,6 +321,18 @@ where
             }
         }
     }
+
+    /// Convert a PyObjectRef to a specific type of PyObject. The caller must
```

Comment:
This comment should be in the trait definition, to make it more conspicuous in rustdoc.

Reply:
Ok, fixed.